crates/dbsp/src/trace/spine_async/mod.rs

```diff
@@ -116,14 +116,27 @@ where
     /// We merge the least recently added batches (ensuring that batches
     /// eventually get merged).
     fn try_start_merge(&mut self, level: usize) -> Option<Vec<Arc<B>>> {
-        // Some LSM implementations do use slightly different numbers per-level.
-        // We didn't see clear benefits with more/slightly less than 12 batches merged at
-        // once or different numbers per level so everything is set to 12 for now.
-        const MAX_BATCHES_MERGED_AT_ONCE: [usize; MAX_LEVELS] =
-            [12usize, 12, 12, 12, 12, 12, 12, 12, 12];
-
-        if self.merging_batches.is_none() && self.loose_batches.len() >= 2 {
-            let n = std::cmp::min(MAX_BATCHES_MERGED_AT_ONCE[level], self.loose_batches.len());
+        /// Minimum and maximum numbers of batches to merge at each level.
+        ///
+        /// Some LSM implementations do use slightly different maximums per
+        /// level.  We didn't see clear benefits with more/slightly less than 12
```

Comment:
This comment no longer seems correct

Reply:
I pushed some comment fixes to a branch `nway-comments`.